src/test/java/com/notably/logic/suggestion/commands/NewSuggestionCommandTest.java

```diff
@@ -0,0 +1,45 @@
+package com.notably.logic.suggestion.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+import com.notably.commons.core.path.AbsolutePath;
+import com.notably.model.Model;
+import com.notably.model.ModelManager;
+import com.notably.model.block.BlockModel;
+import com.notably.model.block.BlockModelImpl;
+import com.notably.model.suggestion.SuggestionModel;
+import com.notably.model.suggestion.SuggestionModelImpl;
+import com.notably.model.viewstate.ViewStateModel;
+import com.notably.model.viewstate.ViewStateModelImpl;
+
+public class NewSuggestionCommandTest {
+    private static AbsolutePath toRoot;
+    private static Model model;
+    private static NewSuggestionCommand newSuggestionCommand;
+
+    @Test
+    public void execute_responseText_test() {
```

Comment:
Take a look at our mod's [test case naming convention](https://nus-cs2103-ay1920s2.github.io/website/coding-standards/java/intermediate.html). I think you need to rename this method